powerspikegg/rawdata/tools/fetch/match.go

```diff
@@ -101,7 +101,11 @@ func (c *matchCommand) ParseFlags(f *flag.FlagSet) error {
 
 // Execute run the command 'match'
 func (c *matchCommand) Execute(ctx context.Context, f *flag.FlagSet, _ ...interface{}) subcommands.ExitStatus {
-	c.ParseFlags(f)
+	parseErr := c.ParseFlags(f)
```

Comment:
Thanks for this fix ;) However, please chain the parseErr like this:
`if err := c.ParseFlags(f); err != nil {`

Avoids availability of `err` in the scope.